### PR TITLE
Rogue newline fix

### PR DESCRIPTION
**Pull request by olsona**
Fixes bug where keys pulled by GoogleAppCredentialStore would have a newline appended, causing auth to fail.

### CODE REVIEW

File: extensions/cloud/portability-cloud-google/src/main/java/org/dataportabilityproject/cloud/google/GoogleAppCredentialStore.java [hunk 1]
```diff
@@ -125,15 +125,19 @@ public AppCredentials getAppCredentials(String keyName, String secretName) throw
 
   private byte[] getRawBytes(String blobName) {
     Bucket bucket = storage.get(bucketName);
-    return bucket.get(blobName).getContent();
+    System.out.println("Blob name: " + blobName);
```
Comment (holachuy) [inline extensions/cloud/portability-cloud-google/src/main/java/org/dataportabilityproject/cloud/google/GoogleAppCredentialStore.java]: Let's remove the System.outs

File: portability-transfer/src/main/java/org/dataportabilityproject/transfer/JobPollingService.java [hunk 2]
```diff
@@ -60,6 +60,8 @@ protected void runOneIteration() throws Exception {
     }
   }
 
+  // TODO: the delay should be more easily configurable
```
Comment (holachuy) [inline portability-transfer/src/main/java/org/dataportabilityproject/transfer/JobPollingService.java]: Do you want this file in this PR or separate?

Comment (olsona) [reply portability-transfer/src/main/java/org/dataportabilityproject/transfer/JobPollingService.java]: Separate, I made an issue #400

File: extensions/cloud/portability-cloud-google/src/main/java/org/dataportabilityproject/cloud/google/GoogleAppCredentialStore.java (hunk 1 above)
Comment (olsona) [reply extensions/cloud/portability-cloud-google/src/main/java/org/dataportabilityproject/cloud/google/GoogleAppCredentialStore.java]: Done

**Pull request merged**